### DISCUSSION TITLE
Filename changes after combine two reports

### DISCUSSION THREAD

**Discussion by ghost** [danielpalme/ReportGenerator#626] [Q&A]
I use the ReportGenerator mainly to combine reports of multiple executables into one.

For this I take the cobertura-coverage-files of the different executables and modify the package-name, afterwards I use ReportGenerator with all the modified files and generate a new cobertura-report.

This works so far except for one thing:
In my XML-file the filenames of the classes are relative
`<class name="TestFile1.h" filename="git\TestFile1.h" line-rate="0.5" branch-rate="0" complexity="0">`
but in the report generated by ReportGenerator there are fix paths
`<class name="TestFile1.h" filename="C:\git\TestFile1.h" line-rate="1" branch-rate="1" complexity="NaN">`

Am I missing some argument? Is this a bug?
The problem afterwards is, that I use this combined report with the Jenkins and there it does not find the files in the path "C:C:\....".

The attached file contains some example-files and a powershell-script to start the whole combine of the reports.
[combineCoverage.zip](https://github.com/danielpalme/ReportGenerator/files/12674483/combineCoverage.zip)

Thank you for your assistance!

**Comment by danielpalme** [comment]
Please have a look at this issue: https://github.com/danielpalme/ReportGenerator/issues/356
